README.md

```diff
@@ -160,4 +160,6 @@ I would like to thank Agaba JohnMark for helping finish this project
 
 ## 📝 License <a name="MIT"> MIT</a>
 
+
 This project is [MIT](./LICENSE) licensed.
+
```

Comment:
As asked by the previous CR your link is still broken and shows a `404 ERROR` when clicked. To resolve this you can replace the following line and verify it:

`This project is [ MIT ]( ./MIT.md ) licensed.`